Makefile

```diff
@@ -15,7 +15,7 @@ JSONNETFMT_ARGS ?= --in-place --pad-arrays
 JSONNET_IMAGE   ?= docker.io/bitnami/jsonnet:latest
 JSONNET_DOCKER  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --entrypoint=jsonnetfmt $(JSONNET_IMAGE)
 
-YAML_FILES      ?= $(shell find . -type f -not -path './vendor/*' -not -path './manifests/*' -not -path './compiled/*' \( -name '*.yaml' -or -name '*.yml' \))
+YAML_FILES      ?= $(shell find . -type f -not -path './vendor/*' -not -path './dependencies/*' -not -path './compiled/*' \( -name '*.yaml' -or -name '*.yml' \))
```

Comment:
Probably better to keep both `./manifests` and `./dependencies`, since the folder might be `manifests` if you run the yaml linting in a Commodore working tree subdirectory `depedencies/lieutenant`.